Updated eclipse instructions and other minor enhancements (#52)
vaadin-learning-center/learning-content: tutorials/getting-started-with-flow/01__how-to-use-vaadin-flow/content.adoc

=== Installing the Development Tools 

* *JDK*: The tutorial uses Java 8. Make sure that you have JDK 8 or later installed and that it is up to date.
** Most Linux distributions can use package managers to install JDK.
** Windows and Mac users need to download JDK from http://www.oracle.com/technetwork/java/javase/downloads/index.html[Oracle's Java SE Downloads].

> * *IDE*: The tutorial uses https://www.eclipse.org[Eclipse], but you can use other IDE that you already master. Make sure you have the latest version installed. 

Reviewer: It is quite complex to find the correck pack navigating from here - https://www.eclipse.org. This - https://www.eclipse.org/downloads/packages/ - leads directly to the correct one

Author: The right corner has a download link. Maybe we should point to https://www.eclipse.org/downloads/ directly. If I have understood it correctly the https://www.eclipse.org/downloads/packages site is kind of legacy thing, now replaced with the installer app.

> * *IDE*: The tutorial uses https://www.eclipse.org[Eclipse], but you can use other IDE that you already master. Make sure you have the latest version installed. 

Reviewer: ```suggestion
* *IDE*: The tutorial uses https://www.eclipse.org[Eclipse], but you can use any other IDE that you familiar with. Make sure you have the latest version installed. 
```
